.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+### Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Type of Change
+
+- [ ] bugfix
+- [ ] feature
+- [ ] enhancement
+- [ ] refactor
+- [ ] style
+- [ ] docs
+- [ ] chore
+
+### Scope
+
+- [ ] frontend
+- [ ] backend
+- [ ] fullstack
+- [ ] infra
+- [ ] tests
+- [ ] UI/UX
+
+### Priority
+
+- [ ] high-priority
+- [ ] medium-priority
+- [ ] low-priority
+
+### Specific Changes
+
+- [ ] variable-rename
+- [ ] linter-fix
+- [ ] prettier-fix
+
+### Dependencies
+
+- [ ] blocked
+- [ ] blocking
+
+### Mandatory Reviewers
+
+@Code-Quality-UI-Specialist
+@Architecture-Design-Reviewer
+@Cloud-Infrastructure-Reviewer
+@Testing-Specialist
+@UI-UX-Reviewer
+@Tech-Leader
```

Comment:
here as well maybe better use GitHub to request
<img width="397" alt="image" src="https://github.com/user-attachments/assets/9dc6d6c9-69d4-4698-9533-4eb262d389e1">